js/templates/guidStillCreatingModal.html

```diff
@@ -0,0 +1,18 @@
+<div class="modal-hero modal-shadow-inner1 bg-dark-blue custCol-primary-darken">
+  <div class="modal-hero-icon"></div>
+  <p class="h6 homeModal-heroText fancy-heading">Welcome</p>
+</div>
+<div class="accordion-window" style="left: 0px;">
+  <div class="accordion-child accordion-active custCol-primary overflowHidden" style="width: 525px; height: 374px;">
+    <div class="bar barTxt h6 custCol-secondary custCol-text">
+      <i class="ion-android-sync spinner"></i>
+      <span class="textOpacity75 fontSize14">Hang tight for a few...</span>
+    </div>
+    <div class="flexContainer">
+      <div class="flexRow">
+        <p class="margin0 padding015 marginBottom20 marginTop20">Your personal keys and OpenBazaar ID are generating and should only take about 30 seconds to finish. You'll be automatically connected to the network once the process is finished, so please hang tight.</p>
+        <p class="margin0 padding015">Looking for something to do in the meantime? Read the <a>OpenBazaar blog</a>.</p>        
```

Comment:
This should have the actual link. Make sure it has the js-externalLink class, so it opens in a browser window instead of in electron.

The link is https://blog.openbazaar.org/